draft-ietf-quic-transport.md

```diff
@@ -1716,11 +1716,12 @@ Note:
   control, which are not expected to be relevant for a closed connection.
   Retransmitting the final packet requires less state.
 
-After receiving a closing frame, endpoints enter a draining period.  An endpoint
+After receiving a closing frame, endpoints enter a draining state.  An endpoint
 that receives a closing frame MAY send a single packet containing a
-CONNECTION_CLOSE frame before entering a draining period, but MUST NOT send
-further packets, which could result in a constant exchange of closing frames
-until the closing period on either peer ended.
+closing frame before entering a draining state, using a CONNECTION_CLOSE frame
+and a NO_ERROR code if no other code is more appropriate.  An endpoint MUST NOT
```

Comment:
nit: since it is possible to send an APPLICATION_CLOSE frame as well, how about simply "using a CONNECTION_CLOSE frame and a NO_ERROR code if appropriate."